monkey/infection_monkey/exploit/wmiexec.py

```diff
@@ -113,33 +133,30 @@ def _exploit_host(self) -> ExploiterResultData:
                 } + build_monkey_commandline(self.servers, self.current_depth + 1)
 
             # execute the remote monkey
+            propagation_timestamp = time()
```

Comment:
Based on our discussion, we should move this timestamp to where the agent binary is copied to the victim machine.